modules/dists/SparseBlockDist.chpl

```diff
@@ -539,6 +549,24 @@ class SparseBlockArr: BaseSparseArr(?) {
     return true;
   }
 
+  proc getLocalSubarray(localeRow, localeCol) const ref {
```

Comment:
I might be missing something, but does this one only apply to 2D Sparse Block where there might be higher dimension ones too? Might it need a where clause or compilerError to check rank == 2 ?

Reply:
Oh, that's a great point and catch…  This should probably be `localeIdx…rank`?  I've definitely been in 2D land, and since CSR/CSC is 2D-specific, lots of my changes have been OK to be, but this leaked out.  Will generalize it, thanks!